### ISSUE TITLE
Logo suggestion

### ISSUE THREAD

**Issue by muuvmuuv** [GitHawkApp/GitHawk#2152]
I just wanted to throw my idea in about the GitHawk logo. I think it would look more modern if we would remove the pr icons around the hawk or make them less present. Most icons and apps nowadays are cleaner in design to not distract the users and I think it would better match with other apps on the home screen. 

Feel free to correct me!

**Comment by schrodincat** [comment]
I think it’s actually really cool looking logo, without PR icons it would be just generic hawk picture.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by muuvmuuv** [comment]
@schrodincat @BasThomas I agree that is a unique identifier. What if we would remove the gradient or make the _git_ less present?

![final icon](https://user-images.githubusercontent.com/11534760/45159855-31c01100-b1e8-11e8-9a85-03fb3516cea8.png)

**Comment by BasThomas** [comment]
Let’s wait for @rnystrom to chime in on this one

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by schrodincat** [comment]
I actually would add gradient to the hawk too, so it would be consistent across the logo

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
I’m open to all of this 😊 but would like to see some shots of the icon on the home screen! That’s the most important to me. 

I like the idea of removing the gradients from the gits, but I really like the shimmer on the hawk! Happy to check out some mockups. Are the icon files in this repo? If not, I’ll upload them when I’m done traveling.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
